src/vr.ts

```diff
@@ -0,0 +1,94 @@
+const SCRIPT_KEYWORD = "    • ";
+
+const scriptGenerator: Fig.Generator = {
+  script: "NO_COLOR=1 vr",
+  postProcess: (out) => {
+    let suggestions: Fig.Suggestion[] = [];
+
+    try {
+      const lines = out.split("\n").filter(Boolean);
+
+      for (const line of lines) {
+        if (!line.startsWith(SCRIPT_KEYWORD)) continue;
+
+        const idx = lines.findIndex((k) => k === line);
+
+        suggestions = [
+          ...suggestions,
+          {
+            name: line.replace(SCRIPT_KEYWORD, ""),
+            description: lines[idx + 1] ? lines[idx + 1].trim() : null,
+          },
+        ];
+      }
```

Comment:
I prefer a C style loop instead of the `.findIndex`.
```suggestion
      for (let i = 0; i < lines.length; i++) {
        const line = lines[i];
        if (!line.startsWith(SCRIPT_KEYWORD)) continue;

        suggestions.push({
          name: line.replace(SCRIPT_KEYWORD, ""),
          ...(lines[i + 1] && { description: lines[i + 1].trim() }),
        }),
      }
```